src/webportal/src/app/job-submission/utils/constants.js

```diff
@@ -97,45 +97,43 @@ export const PROTOCOL_TOOLTIPS = {
   secrets: `Secrets are used to store sensitive data. The value will be masked and won't be seen by other users.`,
   data:
     'Data section is used to generate pre-command that download/mount your data to specific path in container.',
+  dockerImage: 'Please contact admin to make sure which cuda versions in docker image is supported by gpu drivers.',
 };
 
 export const COMMAND_PLACEHOLDER = `'You could define your own Parameters, Secrets or Data mount point on the right sidebar.
 
 All lines will be concatenated by "&&". So do not use characters like "#", "\\" in your command`;
 
 export const DOCKER_OPTIONS = [
-  {
-    key: 'all',
-    text: 'all-in-one+python3.6+gpu (image: ufoym/deepo:all-py36-cu100)',
-    image: 'ufoym/deepo:all-py36-cu100',
-  },
   {
     key: 'tensorflow-gpu-python3.6',
-    text: 'tensorflow+python3.6 with gpu (image: ufoym/deepo:tensorflow-py36-cu100)',
-    image: 'ufoym/deepo:tensorflow-py36-cu100',
+    text: 'tensorflow+python3.6 with gpu, cuda 9.0 (image: ufoym/deepo:tensorflow-py36-cu90)',
+    image: 'ufoym/deepo:tensorflow-py36-cu90',
   },
   {
-    key: 'tensorflow-cpu',
-    text:
-      'tensorflow+python3.6 with cpu (image: ufoym/deepo:tensorflow-py36-cpu)',
+    key: 'tensorflow-cpu-python3.6',
+    text: 'tensorflow+python3.6 with cpu (image: ufoym/deepo:tensorflow-py36-cpu)',
     image: 'ufoym/deepo:tensorflow-py36-cpu',
   },
+  {
+    key: 'tensorflow-gpu-python2.7',
+    text: 'tensorflow+python2.7 with gpu, cuda 9.0 (image: ufoym/deepo:tensorflow-py27-cu90)',
+    image: 'ufoym/deepo:tensorflow-py27-cu90',
+  },
   {
     key: 'tensorflow-cpu-python2.7',
-    text:
-      'tensorflow+python2.7 with cpu (image: ufoym/deepo:tensorflow-py27-cpu)',
+    text: 'tensorflow+python2.7 with cpu (image: ufoym/deepo:tensorflow-py27-cpu)',
     image: 'ufoym/deepo:tensorflow-py27-cpu',
   },
   {
     key: 'pytorch-gpu',
-    text: 'pytorch+python3.6 with gpu (image: ufoym/deepo:pytorch-py36-cu100)',
-    image: 'ufoym/deepo:pytorch-py36-cu100',
+    text: 'pytorch+python3.6 with gpu, cuda 9.0 (image: ufoym/deepo:pytorch-py36-cu90)',
+    image: 'ufoym/deepo:pytorch-py36-cu90',
   },
   {
     key: 'pytorch-cpu',
     text: 'pytorch+python3.6 with cpu (image: ufoym/deepo:pytorch-py36-cpu)',
     image: 'ufoym/deepo:pytorch-py36-cpu',
   },
 ];
-
 export const DEFAULT_DOCKER_URI = 'ufoym/deepo:all-py36-cu100';
```

Comment:
Please change the default docker image as well

Reply:
ok